src/pvrclient-argustv.cpp

```diff
@@ -969,9 +1023,9 @@ PVR_ERROR cPVRClientArgusTV::AddTimer(const PVR_TIMER &timerinfo)
     timerinfo.iClientChannelUid, pChannel->Guid().c_str());
 
   // Try to get original EPG data from ARGUS
-  struct tm* convert = localtime(&timerinfo.startTime);
+  struct tm* convert = localtime((const time_t*) timerinfo.startTime);
   struct tm tm_start = *convert;
-  convert = localtime(&timerinfo.endTime);
+  convert = localtime((const time_t*) timerinfo.endTime);
```

Comment:
I know you're trying to get rid of a compiler warning here. First, you should use c++ style casts  instead of c style casts. Second, a better approach to fix the warning without brute force cast is like I did for pvr.hts: https://github.com/kodi-pvr/pvr.hts/commit/1a0107b3a69e04129bc1ad98d1a578956ea4b10a